nemo_text_processing/text_normalization/de/taggers/tokenize_and_classify.py

```diff
@@ -132,7 +132,7 @@ def __init__(
                 pynini.closure(punct + pynutil.insert(" ")) + token + pynini.closure(pynutil.insert(" ") + punct)
             )
 
-            graph = token_plus_punct + pynini.closure(pynutil.add_weight(delete_extra_space, 1.1) + token_plus_punct)
+            graph = token_plus_punct + pynini.closure((delete_extra_space).ques + token_plus_punct)
```

Comment:
Did you stress test to make sure the lack of a weight is fine?

Reply:
Yes.  It seemed to work fine with whatever I threw at it.